jumpbox/README.md

```diff
@@ -2,6 +2,14 @@
 
 This JumpBox is built off a light-weight AMI which was manually built off of an already configured EC2 instance.
 
+## Prerequisites
+One needs to have jq and aws-cli installed on the machine.
+e.g. for mac,
+```
+brew install awscli
+brew install jq
+```
```

Comment:
install with the AMI ?

Reply:
This is necessary as part of launching the jumpbox. These are already installed inside the jumpbox